instrumentation-api-semconv/src/test/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpServerExperimentalMetricsTest.java

```diff
@@ -79,8 +79,6 @@ void collectsMetrics() {
             metric ->
                 assertThat(metric)
                     .hasName("http.server.active_requests")
-                    .hasDescription(
```

Comment:
Should be `Number of active HTTP server requests.`?

Reply:
I decided to remove assertions on the old metric descriptions -- the stable tests verify all descs though.